src/core/api.js

```diff
@@ -88,6 +88,17 @@ class API {
         }
     };
 
+    exportFilteredCSV = async (query = defaultQuery) => {
```

Comment:
This can be synchronous and could be simplified to:
```JavaScript
getExportUrl = (query) => {
    if (query) {
        const queryKey = this.serializeQuery(
             _.omit(query, ['start', 'limit', 'sortField', 'sortOrder'])
         );
        return `api/export?${queryKey}`;
    } else {
        return `api/export`;
    }
}

Reply:
This way you won't need two functions and can just call `API.getExportUrl()` for exporting all or `API.getExportUrl(query)` for a specific export.

Reply:
I removed the async tag from the function and changed it to the way you suggested. Also removed the exportAll function.